module/dice/action-use-dialog.mjs

```diff
@@ -239,16 +239,15 @@ export default class ActionUseDialog extends StandardCheckDialog {
     const minimizedWindows = [];
     for ( const app of Object.values(ui.windows) ) {
       if ( !app.minimized ) {
-        app.minimize();
         minimizedWindows.push(app);
       }
     }
     for ( const app of foundry.applications.instances.values() ) {
       if ( !app.minimized ) {
-        app.minimize();
         minimizedWindows.push(app);
       }
     }
+    await Promise.all(minimizedWindows.map(app => app.minimize()));
```

Comment:
Maybe let's go with `allSettled` since it's not *critical* that all minimizations succeed, in case one fails for some reason.
```suggestion
    await Promise.allSettled(minimizedWindows.map(app => app.minimize()));
```